build.sbt

```diff
@@ -351,9 +351,18 @@ lazy val compilerBridge: Project = (project in internalPath / "compiler-bridge")
     // compiler instances that are memory hungry
     javaOptions in Test += "-Xmx1G",
     inBoth(unmanagedSourceDirectories ++= scalaPartialVersion.value.collect {
-      case (2, y) if y == 10 => new File(scalaSource.value.getPath + "_2.10")
-      case (2, y) if y >= 11 => new File(scalaSource.value.getPath + "_2.11+")
+      case (2, y) if y == 10            => new File(scalaSource.value.getPath + "_2.10")
+      case (2, y) if y == 11 || y == 12 => new File(scalaSource.value.getPath + "_2.11-12")
+      case (2, y) if y >= 13            => new File(scalaSource.value.getPath + "_2.13")
```

Comment:
For consistency either the match should be `y == 13` or the directory should end in "_2.13+".

Reply:
sure.